java-checks/src/main/java/org/sonar/java/symexec/SymbolicEvaluator.java

```diff
@@ -704,6 +719,12 @@ static PackedStatementStates instantiate() {
       return new PackedStatementStates();
     }
 
+    static PackedStatementStates instantiateWithState(ExecutionState state) {
+      PackedStatementStates result = new PackedStatementStates();
```

Comment:
this should probably be just a wrapper to call the next method (with the list argument)